### PR TITLE
mfem 4.5.2 (new formula)

### PR DESCRIPTION
**Pull request by maxence-wz**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/mfem.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    system "cmake", "--build", "build"
+    system "cmake", "--install", "build"
+    pkgshare.install "examples", "data"
+    # Remove references to Homebrew shims from generated files
+    inreplace lib/"cmake/mfem/MFEMConfig.cmake", HOMEBREW_SHIMS_PATH, ""
+    inreplace pkgshare/"config.mk", HOMEBREW_SHIMS_PATH, ""
+  end
+
+  test do
+    assert_predicate prefix/"lib/libmfem.a", :exist?
```
Comment (chenrui333) [inline Formula/mfem.rb:34]: Need a better test for this.

Comment (maxence-wz) [reply Formula/mfem.rb]: It has been changed.

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
LGTM, thanks @maxence-wz for your first PR to homebrew-core! 🎉

**Comment by maxence-wz** [comment]
@chenrui333 I fixed the issue :
mfem
    * Formula is in wrong path:
        Expected: /home/linuxbrew/.linuxbrew/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/m/mfem.rb
          Actual: /home/linuxbrew/.linuxbrew/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/mfem.rb

**Review by chenrui333 — APPROVED** [review]

**Pull request merged**